src/Navbar.js

```diff
@@ -48,16 +50,13 @@ class Navbar extends Component {
 
     const sidenavLinks = sidenav
       ? sidenav
-      : Children.map(children, (link, index) => {
-          const clonedLink =
-            link && link.props && link.props.id
-              ? React.cloneElement(link, {
-                  ...link.props,
-                  id: `sidenav-${link.props.id}`
-                })
-              : link;
-          return <li key={index}>{clonedLink}</li>;
-        });
+      : [...leftLinks, ...rightLinks].map((link, index) => (
```

Comment:
what would happen to the `Children` ? they would not render in `sidenavLinks`?

Reply:
`Children` are used in line `75` as `link` and should respect `alignLinks` `prop`